src/pages/api/payment.ts

```diff
@@ -0,0 +1,43 @@
+/** @format */
+import Stripe from "stripe";
+
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
+import type { NextApiRequest, NextApiResponse } from "next";
+import { type } from "os";
+const stripeConfig = {
+  apiVersion: "2020-08-27",
+  secretKey: process.env.STRIPE_SECRET_KEY,
+  publicKey: process.env.STRIPE_PUBLIC_KEY,
+};
+const stripe = new Stripe(
+  "sk_test_51MWlLsGBL31qIrQExIrjz7aRXJWxfirJB6ABrOzfsccq55HZ3SNYIlDYP66Jv0pLk7WXq1eJkqaKFUlf0VoRnd0600sG0s3rKS",
```

Comment:
this string is provided by stripe as some sort of api-key? if thats the case we should move it to an env variable